api/rds.proto

```diff
@@ -285,12 +285,20 @@ message RouteAction {
 }
 
 message RedirectAction {
-  // A 302 redirect response will be sent which swaps the host portion of the
-  // URL with this value.
+  // The host portion of the URL will be swapped with this value.
   string host_redirect = 1;
-  // A 302 redirect response will be sent which swaps the path portion of the
-  // URL with this value.
+  // The path portion of the URL will be swapped with this value.
   string path_redirect = 2;
+
+  enum HTTPResponseCode {
+    // Moved Permanently HTTP Status Code - 301.
+    MOVED_PERMANENTLY = 301;
+    // Found HTTP Status Code - 302.
+    FOUND = 302;
+  }
+  // The HTTP status code to use in the redirect response. The default response
+  // code is MOVED_PERMANENTLY (301).
+  HTTPResponseCode response_code = 3;
```

Comment:
IIRC proto will default this to 0, not 301. @htuch would this be covered by the validator?

Reply:
The tests are complaining about the enum needing to start at 0. `api/rds.proto: The first enum value must be zero in proto3.` I'll change the enums to start at 0.